pykokkos/core/cpp_setup.py

```diff
@@ -243,11 +243,16 @@ def invoke_script(self, output_dir: Path, space: ExecutionSpace, enable_uvm: boo
                               compute_capability,   # Device compute capability
                               lib_suffix,           # The libkokkos* suffix identifying the gpu
                               str(compiler_path)]   # The path to the compiler to use
-        compile_result = subprocess.run(command, cwd=output_dir, capture_output=True, check=False)
+
+        if True:
```

Comment:
We will have to think about which condition we want here, as the `script` does not have a unique set of commandline args over all OS that have it. Furthermore, some OS might not even ship it:
- We could switch it on for debug builds ... but it might be something that is helpful independent of the build type we use.
- We could introduce a env variable or a pykokkos internal one to switch it on or off

Reply:
I defaulted it so it only is done in linux systems